bin/cron/form_geos

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env ruby
+#
+# This script incrementally populates the pegasus form_geos table using
+# geolocation of the existing ip_address.
+
+require File.expand_path('../../../pegasus/src/env', __FILE__)
+require 'cdo/db'
+require 'cdo/geocoder'
+
+BATCH_SIZE = 1000
+
+def main
+  time_now = DateTime.now
+  to_geolocate = PEGASUS_DB[:form_geos].where(indexed_at: nil).limit(1000)
```

Comment:
given that this table is currently unpopulated, this limit seems counterproductive.